src/assets/styles.scss

```diff
@@ -0,0 +1,9 @@
+@import './variable';
```

Comment:
You could maybe copy there the content of the same file on the hub: https://github.com/galaxyproject/galaxy-hub/blob/master/src/assets/styles.scss
Then we could be sure to have a similar interface, similar colors

Reply:
Please then add as a comment in the file that was copied from the Hub

Reply:
That should also fix the deployment error